src/contact-us.njk

```diff
@@ -0,0 +1,80 @@
+---
+layout: layouts/nohero.njk
+title: Contact us
+---
+<div>
+  <div class="flex justify-between items-center mb-4 md:px-4 md:-mb-0 md:px-0 md:block">
+    <div class="ff-bg-light product px-4 pb-24 pt-4 md:pt-12 md:p-12 md:grid grid-cols-2">
+      <img class="hidden md:w-72 md:m-auto md:block md:mt-0" src="../images/pictograms/browser_red.png" />
+
+      <form action="https://webto.salesforce.com/servlet/servlet.WebToLead?encoding=UTF-8" method="POST">
+        <input type=hidden name="oid" value="00D8a0000028InL">
+        <input type=hidden name="retURL" value="http://flowforge.com/contact-us?submit=true">
```

Comment:
@joepavitt you should be redirected here after submitting. But let me check on the Salesforce side

Reply:
The form is submitted to Salesforce successfully, but SF then send back a redirect to the page above